src/modules/charge-versions/services/charge-version-workflows.js

```diff
@@ -40,16 +39,14 @@ const getAll = () => service.findAll(chargeVersionWorkflowsRepo.findAll, chargeV
  */
 const getLicenceHolderRole = async chargeVersionWorkflow => {
   const { licenceNumber } = chargeVersionWorkflow.licence;
-  const { startDate } = chargeVersionWorkflow.chargeVersion.dateRange;
+  const startDate = get(chargeVersionWorkflow, 'chargeVersion.dateRange.startDate', null);
```

Comment:
we shouldn't ever have a situation where the charge version start date is null. If this is the case then the UI is at fault

Reply:
You're right - it **shouldn't** happen (ideally)

The `null` is a default value/safety net. Would you prefer that we removed this such that the UI throws an error if any of the objects have a null start date?

Reply:
Yes I think it should throw and error otherwise we might end up with incomplete charge info